scripts/start_server.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
There's a template start server script above, is this necessary?

Reply:
Deleted